set.mm

```diff
@@ -633292,21 +633570,539 @@ fixed reference functional determined by this vector (corresponding to
+      UVKUVLUVMUVOWPVERAXRDXPHFXPAXQDXPHAXCUVEUVMWRQAXPAXOXJUVKUVLUVOWSWTRRAXEX
+      HXJUVJUVIUVLUVOWORAXNXLAXMXKXEHAXMXHUUNIJZXIHFZKFZXKABMXDUBFNZXMUVRUCAYMX
+      DTNZBTNZUDZUUHBXDOPZUEZUEUVSAUWBUWDAYMUVTUWAYTUUBUUAULAUUHUWCUUFAUUHUWCUU
+      FAUUHBUUIOPUWCABBUUCUUCVFAXDUUIBOUUMXAXBUTVPVPBMXDVHVIBXDVJSAUVQXJXHKAUVP
+      XIXIHAUUNBIUURVRQVSRVSVER $.
+  $}
+
+  ${
+    lcmineqlem19.1 $e |- ( ph -> N e. NN ) $.
+    $( Dividing implies inequality for lcm inequality lemma.  (Contributed by
+       metakunt, 12-May-2024.) $)
+    lcmineqlem19 $p |- ( ph -> ( ( N x. ( ( 2 x. N ) + 1 ) ) x.
+    ( ( 2 x. N ) _C N ) )
+    || ( _lcm ` ( 1 ... ( ( 2 x. N ) + 1 ) ) ) ) $=
+      ( c2 cmul co c1 caddc cfz clcmf cfv cn wcel a1i cdvds clcm nnzd syl cgcd
+      cz cbc 2nn nnmulcld peano2nnd nnnn0d 2re nn0ge0d nnge1d lemulge12d bccl2d
+      nnred cr wss cfn fz1ssnn fzfi lcmfnncl lcmineqlem16 lcmineqlem18 remulcld
+      mp2an 1red leadd1dd eqbrtrrd wbr wa jca dvdslcm cmin lcmfunnnd recnd 1cnd
+      simpld pncand oveq2d fveq2d oveq1d eqtrd breqtrrd wceq 2z gcdaddm mp3an13
+      1z addcomd gcd1 eqtr3d lcmineqlem14 ) ABDBEFZGHFZWIBUAFZGWIIFZJKZGWJIFZJK
+      ZCAWIADBDLMAUBNZCUCZUDZABWIWQABCUEZABDABCUKZDULMAUFNZABWSUGADWPUHUIZUJWML
+      MZAWLLUMWLUNMXCWIUOGWIUPWLUQVANZWOLMZAWNLUMWNUNMXEWJUOGWJUPWNUQVANABWICWQ
+      XBURABGHFZWJXFUAFEFWJWKEFWOOABCUSAXFWJABCUDWRABWIGWTADBXAWTUTZAVBXBVCURVD
+      AWMWMWJPFZWOOAWMXHOVEZWJXHOVEZAWMTMZWJTMZVFXIXJVFAXKXLAWMXDQAWJWRQVGWMWJV
+      HRVMAWOGWJGVIFZIFZJKZWJPFXHAWJWRVJAXOWMWJPAXNWLJAXMWIGIAWIGAWIXGVKZAVLZVN
+      VOVPVQVRVSABGSFZBWJSFZGAXRBGWIHFZSFZXSABTMZXRYAVTZABCQZDTMYBGTMYCWAWDDBGW
+      BWCRAXTWJBSAGWIXQXPWEVOVRAYBXRGVTYDBWFRWGWH $.
+  $}
+
+  ${
+    lcmineqlem20.1 $e |- ( ph -> N e. NN ) $.
+    $( Inequality for lcm lemma.  (Contributed by metakunt, 12-May-2024.) $)
+    lcmineqlem20 $p |- ( ph -> ( N x. ( 2 ^ ( 2 x. N ) ) ) <_
+    ( _lcm ` ( 1 ... ( ( 2 x. N ) + 1 ) ) ) ) $=
+      ( c2 cmul co cexp c1 nnred cn0 wcel cr a1i 2re remulcld cn nnmulcld recnd
+      cle wbr caddc cbc cfz cfv 2nn0 nnnn0d nn0mulcld reexpcl mpan syl readdcld
+      clcmf 1red 2nn nn0ge0d nnge1d lemulge12d bccl2d wss fz1ssnn fzfi lcmfnncl
+      cfn mp2an lcmineqlem17 nnrpd lemul2d mpbid mulassd lcmineqlem19 peano2nnd
+      cdvds cz wi nnzd dvdsle syl2anc mpd eqbrtrrd letrd ) ABDDBEFZGFZEFZBWAHUA
+      FZWABUBFZEFZEFZHWDUCFZULUDZABWBABCIZAWAJKZWBLKZADBDJKAUEMABCUFZUGDLKZWKWL
+      NDWAUHUIUJZOABWFWJAWDWEAWAHADBWNANMZWJOAUMUKZAWEABWAADBDPKAUNMZCQZWMABDWJ
+      WPABWMUOADWRUPUQURZIZOZOAWIWIPKZAWHPUSWHVCKXCWDUTHWDVAWHVBVDMZIAWBWFSTWCW
+      GSTABWMVEAWBWFBWOXBABCVFVGVHABWDEFZWEEFZWGWISABWDWEABWJRAWDWQRAWEXARVIAXF
+      WIVLTZXFWISTZABCVJAXFVMKXCXGXHVNAXFAXEWEABWDCAWAWSVKQWTQVOXDXFWIVPVQVRVSV
+      T $.
+  $}
+
+  ${
+    lcmineqlem21.1 $e |- ( ph -> N e. NN ) $.
+    lcmineqlem21.2 $e |- ( ph -> 4 <_ N ) $.
+    $( The lcm inequality lemma without base cases 7 and 8.  (Contributed by
```

Comment:
What are the "base cases 7 and 8"?

Reply:
In the next PR. I don't know when I have time to work on it and I want it merged so that I don't have to worry about future merge conflicts. This merge conflict I had on this PR has cost me over 1 hour.
I wanted to have smaller PR's but this time it sadly wasn't possible. The next PR's from me should have a few theorems and not 20 :)